docs/glossary.md

```diff
@@ -186,6 +228,12 @@ A potential function call to be voted on in a referendum. Proposals modify the b
 
 A system of rules that allows two or more entities of a communications system to transmit information. The protocol defines the rules, syntax, semantics and synchronization of communication and possible recovery methods.
 
+## Random Seed
+
+A random seed is psuedo-random number available on-chain. It is used in various
```

Comment:
```suggestion
A random seed is pseudo-random number available on-chain. It is used in various
```

Reply:
Besides the "pseudo" typo, a seed itself should not be pseudorandom, but from an actual randomness source.  I'm not sure if the seed counts as actual randomness or not. @hndnklnc ?